edx/frontend-app-payment
# Updates for new logo for rebranding

[REV-1604](https://openedx.atlassian.net/browse/REV-1604). 
This is part of the rebranding effort, the styles were already updated in https://github.com/edx/frontend-app-payment/pull/457. 

This PR handles getting the page ready for the logo update (via frontend-component-header). And removing the no longer needed locally sourced favicon image. Pls note the favicon itself in the Payment MFE was not in index.html but I've added it. 

To test this branch locally:
`npm install --save @edx/brand@npm:@edx/brand-edx.org`
`npm start`

Please note you will not see the new logo yet - but if you inspect the DOM and search for: 
`<img class="d-block" src="https://edx-cdn.org/v3/default/favicon.ico" alt="edX">`
